.rubocop.yml

```diff
@@ -59,3 +59,6 @@ Style/Documentation:
   Exclude:
     - 'lib/**/*'
     - 'spec/**/*'
+Lint/ConstantDefinitionInBlock:
+  Exclude:
+    - 'spec/**/*'
```

Comment:
Even if it’s just for testing, defining constants inside a block can cause mistakes, so I’d like to avoid it. Please don’t exclude this.